commitlint.config.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * @type {import('@commitlint/types').UserConfig}
+ */
+const config = {
+  extends: ['@commitlint/config-conventional'],
+  rules: {
+    'subject-case': [
+      0,
+      'always',
+      ['sentence-case', 'start-case', 'pascal-case', 'upper-case'],
+    ],
+  },
+};
+
+module.exports = config;
```

Comment:
Looks like commitlint supports ESM?

```suggestion
/**
 * @type {import('@commitlint/types').UserConfig}
 */
export default {
  extends: ['@commitlint/config-conventional'],
  rules: {
    'subject-case': [
      0,
      'always',
      ['sentence-case', 'start-case', 'pascal-case', 'upper-case'],
    ],
  },
};
```

Reply:
Or use yaml. I usually find that the rule's objects in these types of config files aren't completely typed anyways, and you have to look at the docs anyways lol.

Reply:
> Or use yaml. I usually find that the rule's objects in these types of config files aren't completely typed anyways, and you have to look at the docs anyways lol.

I'll try to figure this out either tonight or Monday

Reply:
Done in https://github.com/wxt-dev/wxt/pull/1385/commits/6d3d977f9557538076a71773255d296b1c52770f